tests/e2e/specs/in-game/one-offs/4_fours.spec.js

```diff
@@ -98,7 +99,6 @@ describe('FOURS', () => {
     });
 
     it('Prevents playing a 4 when opponent has no cards in hand', () => {
-      cy.skipOnGameStateApi();
```

Comment:
Same idea as with threes. Can now run this in the gamestate api, now that the error messages are normalized.